lms/djangoapps/teams/static/teams/js/views/team_count_detail.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * View for displaying the number of teams in a topic.
+ */
+;(function (define) {
+    'use strict';
+    define(['backbone', 'text!teams/templates/team_count_detail.underscore'], function (Backbone, template) {
+        var TeamCountDetailView = Backbone.View.extend({
+            tagName: 'p',
+            className: 'team-count',
+
+            initialize: function () {
+                this.template = _.template(template);
```

Comment:
Nit: in general we declare the template at the same level as `initialize` and `render` in the object passed into `BackbonView.extend`.